Ad9208 prepare

stefpopa opened the pull request:
This PR comes before adding support for the AD9208

mirceacaprioru commented on projects/drivers/jesd204/axi_adxcvr.c:
```diff
@@ -219,39 +214,47 @@ int32_t adxcvr_clk_set_rate(struct adxcvr *xcvr,
 	}
-	if (ret < 0)
-		return ret;
+
```
nitpick: extra line

dbogdan commented on projects/drivers/jesd204/axi_adxcvr.c:
```diff
@@ -80,9 +80,8 @@
 #define ADXCVR_DRP_PORT_ADDR_COMMON		0x00
```
We should state the fact that the BROADCAST feature will write the same value to all channels. Since some times not all channels are configured the same (e.g., clock source is different), an update of some register's bits should be made independently for each channel to not overwrite other settings. The BCAST read might also fail since it returns the ORed between all channels.

dbogdan commented on projects/drivers/util/util.c:
```diff
@@ -158,3 +158,21 @@ uint64_t do_div(uint64_t* n,
+uint64_t div64_u64_rem(uint64_t dividend, uint64_t divisor, uint64_t *remainder)
+{
+	*remainder = dividend % divisor;
+	return dividend / divisor;
```
For consistency with the rest of the file I would add a new line.

dbogdan replied:
... and for the same reason, please add a short comment before the implementation.

stefpopa commented:
Changes in v2:
* Added a more explicit commit message for 5df46a3064623084d202101f96ec4a933eb59476
* Added a new line before returning
* Added comments for the implementation of the util helper functions

stefpopa commented:
Will merge later if there are no objections.
